### ISSUE TITLE
Error in SIRD_SetReadings

### ISSUE THREAD

**Issue by git-developer** [mumpitzstuff/fhem-SIRD#2]
I'm using fhem-SIRD to control a Medion MD 86062 internet radio.

Occasionally, I find the following message in the log:
```
2018.10.17 08:21:54 3: sird_md86062: SetReading failed (please report this bug).
HASH(0x4405578)

Error: Not a HASH reference at ./FHEM/17_SIRD.pm line 1612.
```
Line 1612:
```
  elsif (('sys.audio.mute' eq $nodeName) && exists($node->{value}->{u8}{value}))
```

The radio was switched off when the message was logged.

**Comment by mumpitzstuff** [comment]
Some other users reported similar problems (FHEM forum) in some rare conditions. The reason is unknown at the moment. The $xml variable seems to be empty somehow but some checks like exists($xml->xyz) seems to be true.
It may be that i have to enhance the check with some type checks... I will hopefully fix this within the next days.

**Comment by mumpitzstuff** [comment]
Should be fixed by: 00ea6814103c0b74822945cae831bb47bdfdceb6
